_posts/meeting-notes/2020-03-18-all-hands.md

```diff
@@ -0,0 +1,220 @@
+            - can we pause for 1/2 mths
+            - uv: concerned with business continuity risks. what if a number of us disappear for a period of time? Are we in a position where we could go on pause for a number of months?
+                - bl: org, or project? uv: both
+                    - uv: not concerned with existing client, directly
+        - priorities?
+            - bl: member safety/comfort; org public role; individual client relations/loss of business
+                - bl: do we turn down or hard-no certain opportunities
+                    - dc: union-speak: "right to refuse unsafe work"
+                    - bl: "unsafe" might have community consequences, not just worker safety
+                    - can hypha tell someone "no don't take this opportunity as hypha"
+                    - yj: as directors, what are our responsibilities to employees
+                    - bl: not simple, as our work can enable unsafe community activity
+                    - may be harming short-term financial health of members and org
+                    - uv: +1 for granting Hypha the ability to block member activities if they are considered unsafe for either members or the community at large, but tightly scoped to COVID-19 (+1 el, dc, bl, pc)
+    - Externally: Hypha-to-World
+        - [WIP pad](https://hackmd.io/D_EFCPkpTeKyAuYJ_C-3Cw?view)
+        - bl: goals?
+          - bl wrote: https://github.com/hyphacoop/organizing/issues/229#issuecomment-600351763
+          - uv: not ready to state beyond what's in doc
+          - dc: mutual aid support for ppl > businesses
+              - COVID response doesn't do enough to individuals
+                  - does direct giving, but not enough
+          - pc: eager to help people with casual/informal supports (how to hang out), as feel intertwined with work health
+          - pc: existing projects are ramping up and many need support. what would a hypha "soup-kitchen day" feel like?
+        - next steps:
+          - elon and garry and ben will work on website
+          - scheduled next 3 meetings
+          - is BBB ready for use?
+              - https://bbb.hypha.coop
+          - dc: would love to use BBB internally. academics saying nice things about breakout room modules
+          - bl: proposal, using it for next bizdev meeting? +1 uv
+              - el: can do.
+              - #todo:el remove signup restrictions from bbb
+
+- AGM in the time of COVID-19
+    - Options
+        1. business-as-usual: wait. make go-no-go decision May 15. fallback plan if needed
+        2. postpone AGM: can wait up to 18mo
+        3. go full remote. figure out how to do social thing.
+            - concern of effectiveness
+            - maybe should be in person for first to approve bylaws
+        4. hybrid. (directors in person?)
+            - risk: ppl have relationships with high-risk ppl
+
+- "business as usual" but select plan B: postpone slightly if situation allows / "full remote"
```

Comment:
@patcon I think we have a decision on this, but we seem to be missing a note about the decision. Is this line the decision and we are just missing the tag?

cc. @dcwalk

Reply:
```suggestion
- "business as usual" but select plan B: postpone slightly if situation allows / "full remote"
- **DECISION**:
  - May 15 to decide go/no go for in-person
  - second option: shift to june
  - third option: full remote
```

Reply:
looks great!that is what we agreed to